SendWithUs.Client/Requests/SendRequestConverter.cs

```diff
@@ -128,21 +141,21 @@ protected void WriteRecipientsList(JsonWriter writer, JsonSerializer serializer,
             foreach (var address in recipients)

```

Comment:
Sure, that's the serialization part, but I'm just saying you call them `recipients` everywhere else, so when you get one of them here in this loop, why decide to call it an `address` now? It's still technically a `recipient`, you're just about to serialize it as an `address`. Again, don't care *that much*, just wanted to explain my thinking. :smile: